test/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+#
+# Distributed Linear Algebra with Future (DLAF)
+#
+# Copyright (c) 2018-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+set(DLAF_FORTRAN_MPIRUN "" CACHE STRING "MPI run command.")
+separate_arguments(_DLAF_FORTRAN_MPIRUN UNIX_COMMAND ${DLAF_FORTRAN_MPIRUN})
+
+add_subdirectory(utils)
+add_subdirectory(fpp)
+
+set(TEST_EXECUTABLES
+    init
+    pspotrf pdpotrf pcpotrf pzpotrf
+    pssyevd pdsyevd pcheevd pzheevd
+    pssygvx pdsygvx pchegvx pzhegvx
+)
+
+foreach(TEST ${TEST_EXECUTABLES})
+    add_executable(${TEST} ${TEST}.f90)
+    target_link_libraries(${TEST} PRIVATE 
+        DLAF_Fortran 
+        testutils
+        pxpotrf
+        pxheevd
+        pxhegvx
```

Comment:
Just checking: these are just "supports" for the test, right? In principle (it's not a request, just checking my understanding), these should be added as dependency just for the related target, right? (I mean, `p?potrf` tests do not need `pxheevd` and `pxhegvx` as dependency).

Reply:
Yes, they are not required. I think I was a bit lazy here initially, I'll change it.